definitions/variable/emissions/tag_land-removal-options.yaml

```diff
@@ -0,0 +1,16 @@
+- Land Removal Option:
+    - Agroforestry:
+        description: agroforestry
```

Comment:
Maybe @flohump can provide a bit more text here to describe agroforestry

Reply:
"agroforestry intentionally integrate trees with crops or pasture on the same land"